khal/controllers.py

```diff
@@ -21,6 +21,8 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
+from __future__ import unicode_literals
+
```

Comment:
The linked website only mentions a few issues in detail, and I don't see them applying to us.

> Because only some modules have that future-import, you can't know whether a particular string literal produces a unicode or bytestring.

Without this import, literals are `str`, *not* bytestrings. With it, they become unicode strings (`unicode`). And stdlib still works with unicode strings, eg:

    >>> with open(u'test', u'w') as f:
    ...   f.write("1")

It's not merely readability either, IMHO. When writing new code, you don't need to remember to add adding `u'` to new strings - and forgetting that may cause subtle unexpected error.

Personally, I don't see drawbacks with using `unicode_literals` on khal's codebase, nor any way this can come back and bite us in a forseeable future, so I'd much rather keep it.